tm-bench/main.go

```diff
@@ -132,6 +132,22 @@ func calculateStatistics(client tmrpc.Client, minHeight int64, timeStart, timeSt
 		os.Exit(1)
 	}
 
+	maxHeight := info.LastHeight
+	diff := maxHeight - minHeight
+
+	blockMetas := info.BlockMetas
+	offset := len(blockMetas)
+	for len(blockMetas) < int(diff) {
+		// get blocks between minHeight and last height
+		info, err := client.BlockchainInfo(minHeight+int64(offset), 0)
```

Comment:
Interestingly, this returns (last block - 20 to last block), not (first block to first block + 20). This logic assumes the latter.